src/test/java/org/walkmod/javalang/compiler/TypeVisitorAdapterTest.java

```diff
@@ -466,39 +466,6 @@ public void testGenericMethodOverload() throws Exception {
 				"public static void A.f1(java.util.Collection)");
 	}
 
-	/**
-	 * This kind of method is extensively used in test frameworks like e.g. EasyMock.
-	 */
-	@Test
-	public void testUnboundGenericMethodReturn() throws Exception {
```

Comment:
The code does not pass javac. I confused things when providing it. Since it was just an expression parse it got unnoticed.